test/resources/verify.go

```diff
@@ -65,17 +65,18 @@ func KSOperatorCRVerifyConfiguration(t *testing.T, clients *test.Clients, names
 	// Delete a single key/value pair
 	verifySingleKeyDeletion(t, LoggingConfigKey, loggingConfigMapName, clients, names)
 
+	// Verify HA config
+	VerifyHADeployments(t, clients, names)
+
 	// Use an empty map as the value
 	verifyEmptyKey(t, DefaultsConfigKey, defaultsConfigMapName, clients, names)
 
 	// Now remove the config from the spec and update
 	verifyEmptySpec(t, loggingConfigMapName, clients, names)
-
-	// Verify HA config
-	VerifyHADeployments(t, clients, names)
 }
 
 func verifyDefaultConfig(t *testing.T, ks *v1alpha1.KnativeServing, defaultsConfigMapName string, clients *test.Clients, names test.ResourceNames) {
+	configureIngressClass(ks)
```

Comment:
I think it is better to call `configureIngressClass()` in `getTestKSOperatorCRSpec()`. The `getTestKSOperatorCRSpec` sets the `ks.spec` for tests so we can align with it.

Reply:
yeah, I agree. The feedback has been reflected. Thanks.